nrcan_etl/src/energuide/embedded/evaluation_type.py

```diff
@@ -5,12 +5,15 @@
 class EvaluationType(enum.Enum):
     PRE_RETROFIT = 'D'
     POST_RETROFIT = 'E'
+    INSENTIVE_PROGRAM = 'F'
```

Comment:
Does "INSENTIVE" stand for something in particular? Is it possibly INCENTIVE?

Reply:
Good catch, spelling is my weak point

Reply:
I became a programmer to avoid spelling tbh